src/isar/state_machine/states/cancel.py

```diff
@@ -53,4 +53,10 @@ def start(self):
         self.state_machine.to_next_state(next_state)
 
     def stop(self):
-        self.logger.info(f"State transitions: {self.state_machine.transitions_list}")
+        state_transitions_list: list = [
+            f"\n  {transition}" if (i + 1) % 10 == 0 else f"{transition}"
+            for i, transition in enumerate(list(self.state_machine.transitions_list))
+        ]
+
+        state_transitions: str = ", ".join(state_transitions_list)
+        self.logger.info(f"State transitions:\n  {state_transitions}")
```

Comment:
What about setting this as a separate function? `log_state_transitions`.

Then the stop function doesn't have to have complex arithmetics directly within it.

Reply:
Good idea, will do.